frontend/lindsay-wildlife-treasure-hunt-frontend/src/Components/GameDisplay/GameDisplay.css

```diff
@@ -145,8 +144,119 @@
 
 /* Manual Entry Display */
 
+.manual-entry-container {
+    align-items: center;
+    justify-content: center;
+    display: inline-block;
+    font-family: 'Stag';
+}
+
+.numpad-body {
+    margin-left: auto;
+    margin-right: auto;
+    height: 95vw;
+    width: 65vw;
+    display: grid;
+    grid-template: 1.2fr 1fr 1fr 1fr 1fr / 1fr 1fr 1fr;
+    grid-gap: 5px;
+    max-height: 500px;
+    max-width: 450px;
+    padding: 17px;
+}
+
+.man-btn {
+    border-style: none;
+    border-radius: 100%;
+    font-family: 'Stag';
+    font-size: 34px;
+    background-color: #99CC66;
+    color: white;
+}
+
+.display {
+    color: #303023 !important;
+    font-size: 48px;
+    grid-row: 1;
+    grid-column: 1/4;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-right: 5px;
+    padding-left: 5px;
+    overflow:hidden;
+    margin-bottom: 5px;
+    border: 1px solid;
+    border-radius: 10px;
+}
+
+/* buttons for manual entry */
+
+.btn-del {
```

Comment:
hm. Im wondering if its possible to build the grid without hardcoding each like this. Maybe we can use HTML tables?

Reply:
That's the only way I know to use the CSS Grid Display type - hardcoding the location of each button. I am less familiar with using HTML tables, but if you think it would be a better approach, I can look into it.

Reply:
nah its ok